library/Garden/ArrayContainer.php

```diff
@@ -44,8 +44,7 @@ private function normalizeID($id) {
      *
      * @param string $id Identifier of the entry to look for.
      *
-     * @throws NotFoundException  No entry was found for this identifier.
-     * @throws ContainerException Error while retrieving the entry.
+     * @throws ContainerNotFoundException  No entry was found for this identifier.
```

Comment:
The previous two hints were pointing at a class that didn't exist. I put the correct class in here.